deployment/enterprise/.env.template

```diff
@@ -175,9 +175,6 @@ ACCOUNT_EXPIRATION=1
 
 # for links generated in emails
 
-#MERGIN_BASE_URL=http://localhost:5000
-MERGIN_BASE_URL=fixme
```

Comment:
hm, we need to make sure this make sense and is always set - maybe we should remove default value then